platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -16,10 +22,12 @@ + (instancetype)sharedManager {
     void (^setupBlock)(void) = ^{
         dispatch_once(&onceToken, ^{
             _sharedManager = [[self alloc] init];
+            _sharedManager.sessionConfiguration = nil;
         });
     };
     if (![[NSThread currentThread] isMainThread]) {
-        dispatch_sync(dispatch_get_main_queue(), ^{
+        // This is a temporary fix to avoid a dead lock
```

Comment:
This will be removed once this https://github.com/mapbox/mapbox-gl-native/pull/13754 lands.